cloudnet/src/main/resources/lang/dutch.properties

```diff
@@ -0,0 +1,397 @@
+#
+# command create
+command-create-start=De services worden gemaakt ...
+command-create-by-task-success=De services zijn gemaakt op basis van de Task. Deze kunnen worden beheerd met het commando 'service'
+command-create-by-task-failed=De services kunnen niet worden gemaakt
+command-create-by-task-not-found=Deze Task bestaat niet
+command-create-new-service-success=De services zijn met succes gemaakt. Deze kunnen worden beheerd met het commando 'service'
+# opdrachtscherm
+command-screen-enable-for-service=Het automatisch loggen van het consolelogboek voor de service %uniqueId%:%name% is geactiveerd
+command-screen-disable-for-service=Het automatisch loggen van het consolelogboek voor de service %uniqueId%:%name% is gedeactiveerd.
+command-screen-write-success=Het commando "%command%" is succesvol verzonden naar de services %targets%
+command-screen-write-no-screen=U bevindt zich op geen enkel scherm
+command-screen-list-no-screen=U bevindt zich op geen enkel scherm
+command-screen-list=Actieve schermen:% schermen%
+command-screen-disabled=Alle schermen zijn uitgeschakeld: %screen%
+# command tasks/groups
+command-service-base-add-template-success=De sjabloon is gevonden en is toegevoegd aan de configuratie
+command-service-base-add-inclusion-success=De opname is succesvol toegevoegd aan de configuratie
+command-service-base-add-deployment-success=De implementatie is succesvol toegevoegd aan de configuratie
+command-service-base-add-group-no-group-found=Deze groep bestaat niet
+command-service-base-remove-template-success=De sjabloon is succesvol verwijderd uit de configuratie
+command-service-base-remove-inclusion-success=De opname is succesvol verwijderd uit de configuratie
+command-service-base-remove-deployment-success=De implementatie is succesvol verwijderd uit de configuratie
+command-service-base-remove-jvm-option-not-found=Deze JVM-optie bestaat niet voor deze configuratie
+command-service-base-remove-jvm-option-success=De JVM-optie is succesvol verwijderd
+command-service-base-remove-process-parameter-option-success=Deze procesparameter is succesvol verwijderd
+command-service-base-remove-process-parameter-option-not-found=Deze procesparameter bestaat niet voor deze configuratie
+command-service-base-add-jvm-option-already-existing=Deze configuratie heeft deze JVM-optie al
+command-service-base-add-jvm-option-success=De JVM-optie is succesvol toegevoegd aan de configuratie
+command-service-base-add-process-parameters-option-already-existing=Deze configuratie heeft al deze procesparameters
+command-service-base-add-process-parameters-option-success=De procesparameter is succesvol toegevoegd aan de configuratie
+command-service-base-clear-jvm-options-success=De JVM-opties voor deze configuratie zijn met succes gewist
+command-service-base-group-not-found=Deze groep bestaat niet!
+command-service-base-create-group=De lege groep is succesvol aangemaakt. Configureer deze met het commando 'tasks' of in het configuratiebestand
```

Comment:
Tasks command to manage groups?

Reply:
Isn't that also the case in the German
![image](https://user-images.githubusercontent.com/74465789/125949776-6ba052ef-cb19-4fc3-a6b7-d64dda299d65.png)
 version?

Reply:
Hmm okay, so this is fine